muselog/flask.py

```diff
@@ -0,0 +1,46 @@
+"""
+Custom logger for user auth service in python flask 
+"""
+
+import sys
+import logging
+
+from time import strftime
+from muselog import add_gelf_handler
+
+logger = logging.getLogger(__name__)
+
+# http://flask.pocoo.org/docs/0.12/api/#flask.Flask.after_request
+def log_request(request_duration, request, response):
```

Comment:
Should typehint these.

Reply:
No type hint annotations for request object @ian
https://github.com/requests/requests/issues/3855

Do I just go ahead with `Any`
An alternative would be to use  pip install `werkzeug` and use it for type hint
E.g.
```
from werkzeug.wrappers import Request, Response
def log_request(request_duration: int, request: Request, response: Response) -> Response:
```

Reply:
Or 
```
def log_request(request_duration: int, request: Any, response: Any) -> Any:
```

Reply:
@ian-axelrod still waiting on your input here